spec/rubocop/cop/style/if_unless_modifier_spec.rb

```diff
@@ -6,7 +6,11 @@
   subject(:cop) { described_class.new(config) }
 
   let(:config) do
-    RuboCop::Config.new('Layout/LineLength' => line_length_config)
+    RuboCop::Config.new('Layout/LineLength' => line_length_config,
+                        'Layout/IndentationStyle' => {
+                          'Enabled' => true,
```

Comment:
I would expect the test to fail if `'Enabled' => false` and `'EnforcedStyle' => 'spaces'`, but the test passes unexpectedly. Perhaps the changes to lib/rubocop/cop/mixin/statement_modifier.rb may not be enough.

Reply:
My thinking, when I made the change, was that `Style/IfUnlessModifier` needs to find out how many spaces each tab corresponds to when it finds indentation with tabs. But I see now that I haven't really followed though on this idea. 

I still believe that it should not matter to `Style/IfUnlessModifier` if 'Layout/IndentationStyle' is enabled or not when it tries to calculate the total line length of a line that is indented with tabs, but I'll try to come up with something better.